### PR TITLE
Removed feedback section and unnecesary fields.

### PR DESCRIPTION
**Pull request by JaviCandia**
- **This is the new look**
image

- Removed house number field in the shipping section
image

- Removed the form in payment section
image

### CODE REVIEW

File: components/organisms/o-order-confirmation.vue [hunk 1]
```diff
@@ -55,26 +55,7 @@
...
-        class="sf-heading--left"
-      />
-      <p class="paragraph">
-        {{ $t('Your feedback is important for us. Let us know what we could improve.') }}
-      </p>
-      <textarea
-        class="feedback"
-        v-model="feedback"
-        :placeholder="$t('Type your opinion')"
-      />
```
Comment (alanboy) [inline components/organisms/o-order-confirmation.vue:70]: don't remove it. please simply comment it out with and add this at the top:
```
<!-- Not to use in V2 -->
```

**Review by alanboy — CHANGES_REQUESTED** [review]
I also don't see the changes to address and making the House/apartment not required.

**Comment by JaviCandia** [comment]
> I also don't see the changes to address and making the House/apartment not required.

@alanboy Look closely about whats is happening with the comments in o-payment.vue and o-shipping.vue

File: components/organisms/o-shipping.vue [hunk 2]
```diff
@@ -47,7 +47,7 @@
         :error-message="$t('Field is required')"
         @blur="$v.shipping.streetAddress.$touch()"
       />
-      <SfInput
+      <!-- <SfInput
```
Comment (alanboy) [inline components/organisms/o-shipping.vue:50]: we dont want to hide this, we want to make it not-required.

File: components/molecules/m-cookie-notification.vue [hunk 3]
```diff
@@ -4,7 +4,7 @@
       <div class="cookie">
         <div class="cookie__message">
           {{ message }}
-          <a
```
Comment (alanboy) [inline components/molecules/m-cookie-notification.vue:7]: this file has just a space, can you undo the changes?

**Review by alanboy — CHANGES_REQUESTED** [review]

File: components/organisms/o-shipping.vue (hunk 2 above)
Comment (kaitco13) [reply components/organisms/o-shipping.vue:50]: we don’t currently save apt number anywhere though. Would you want to add it onto the end of street or something?

Comment (alanboy) [reply components/organisms/o-shipping.vue:50]: oh, is that so? in that case, then yes. removing it makes sense OR append it to our street. thoughts? i think its very common to have this field.

Comment (JaviCandia) [reply components/organisms/o-shipping.vue:50]: Attaching the house number to the street would be an extremely easy option to do.

Comment (alanboy) [reply components/organisms/o-shipping.vue:50]: OK fine, but please open an issue to have a second address line in apicur.io.

Comment (alanboy) [reply components/organisms/o-shipping.vue:50]: and after you do, please add a link to the issue here to the comment (in the code).

File: components/molecules/m-cookie-notification.vue (hunk 3 above)
Comment (JaviCandia) [reply components/molecules/m-cookie-notification.vue:7]: Sure.

File: components/organisms/o-shipping.vue (hunk 2 above)
Comment (JaviCandia) [reply components/organisms/o-shipping.vue:50]: So... you mean something like this?
image

Comment (JaviCandia) [reply components/organisms/o-shipping.vue]: I think it's done.

File: components/organisms/o-shipping.vue [hunk 4]
```diff
@@ -47,15 +47,12 @@
...
+      <!-- https://github.com/building-21/pos-erp/issues/938 -->
       <SfInput
         v-model.trim="shipping.apartmentNumber"
         class="form__element"
         name="apartment-number"
         :label="$t('House/Apartment number')"
-        :required="true"
-        :valid="!$v.shipping.apartmentNumber.$error"
-        :error-message="$t('Field is required')"
-        @blur="$v.shipping.apartmentNumber.$touch()"
```
Comment (alanboy) [inline components/organisms/o-shipping.vue:58]: why remove this @blur and the validation ?

File: components/organisms/o-shipping.vue [hunk 5]
```diff
@@ -199,10 +196,6 @@ export default {
         required,
         unicodeAlphaNum
       },
-      apartmentNumber: {
-        required,
```
Comment (alanboy) [inline components/organisms/o-shipping.vue:203]: can we not remove the required and leave the unicodeAlphaNum validation?

File: components/organisms/o-shipping.vue [hunk 6]
```diff
@@ -47,15 +47,12 @@
         :error-message="$t('Field is required')"
         @blur="$v.shipping.streetAddress.$touch()"
       />
+      <!-- https://github.com/building-21/pos-erp/issues/938 -->
```
Comment (alanboy) [inline components/organisms/o-shipping.vue:50]: Since we are doing this at the adapter level, please remove this comment from here ( I originally thought you would do the mapping in VSF) and add it here: 
https://github.com/building-21/pos-erp/pull/931/files/85a517ccd2becd1991d4a3e2d66f2fbfbdfad641#r625486188

**Review by alanboy — CHANGES_REQUESTED** [review]

File: components/organisms/o-shipping.vue (hunk 4 above)
Comment (JaviCandia) [reply components/organisms/o-shipping.vue:58]: Aren't we supposed to not want it to be a required field?

By removing those properties we remove the hint 
image

File: components/organisms/o-shipping.vue (hunk 5 above)
Comment (JaviCandia) [reply components/organisms/o-shipping.vue:203]: Even if we only leave the unicodeAlphaNum validation, we will not be able to click on "continue to payment" if we leave the apartment number field empty.

image

Comment (JaviCandia) [reply components/organisms/o-shipping.vue:203]: In fact, you can see that PhoneNumber is the only field that is not required in the entire form, that field doesn't have any of the properties that I deleted in apartmentNumber

File: components/organisms/o-shipping.vue (hunk 6 above)
Comment (JaviCandia) [reply components/organisms/o-shipping.vue:50]: sure.

**Review by alanboy — APPROVED** [review]
